src/app/pages/MediaArticlePage/PagePromoSections/LatestMediaSection/index.styles.tsx

```diff
@@ -0,0 +1,49 @@
+import pixelsToRem from '#app/utilities/pixelsToRem';
+import { Theme, css } from '@emotion/react';
+
+const styles = {
+  LatestMediaSection: ({ mq, isDarkUi, palette }: Theme) =>
```

Comment:
This should start with a lower case, for consistency with other styles. 
```suggestion
  latestMediaSection: ({ mq, isDarkUi, palette }: Theme) =>
```